lib/src/equatable_utils.dart

```diff
@@ -47,3 +47,6 @@ int _finish(int hash) {
   hash = hash ^ (hash >> 11);
   return 0x1fffffff & (hash + ((0x00003fff & hash) << 15));
 }
+
+String toStringWithProps(Type runtimeType, List<Object> props) =>
```

Comment:
Might want to have a check for is props is null. Also thoughts on renaming the method to mapPropsToString?